contents/tutorials/abn-testing.md

```diff
@@ -0,0 +1,88 @@
+
+## Creating an A/B/n test
+
+To create an A/B/n, go [experiments tab in PostHog](https://app.posthog.com/experiments) and click "New experiment." Add a name, feature flag key (I choose `homepage-abn`), and description if you want. Under "Experiment variants," click "Add test variant" twice to add two new variants. Rename the three variants if you want. 
+
+![A/B/n experiment](../images/tutorials/abn-testing//experiment.png)
+
+Fill out the rest of your details like participants, goal type, secondary metrics, and minimum acceptable improvement. Once done, click "Save as draft" and we will go implement the A/B/n test in our app.
+
+## Implementing our A/B/n test
+
+Implementing the A/B/n requires checking the feature flag variant and handling each of our cases. We will use a [Next.js](/docs/libraries/next-js) app where we already set up the `[PostHogProvider](/docs/libraries/next-js#app-router)` for this.
+
+> Need a full guide for creating a Next.js app? Read our [How to set up Next.js A/B tests](/tutorials/nextjs-ab-tests) tutorial.
+
+In our a `page.js` file, we: 
+
+1. Import `useFeatureFlagVariantKey` from `posthog-js/react`, and `useState` and `useEffect` from `react`
+2. Set up a state for `mainCopy` using `useState`. 
+3. Set the`mainCopy` state in a `useEffect` using the variant key value we get from PostHog.
+4. Show the `mainCopy` state to users in our component.
+
+Together, this looks like this:
+
+```js
+// app/page.js
+'use client'
+import { useFeatureFlagVariantKey } from 'posthog-js/react'
+import { useState, useEffect } from 'react'
+
+export default function Home() {
+
+  const [mainCopy, setMainCopy] = useState('Welcome to our app')
+  const copyVariant = useFeatureFlagVariantKey('homepage-abn')
+  
+  useEffect(() => {
+    if (copyVariant && copyVariant !== 'control') {
+      setMainCopy(`Welcome to our ${copyVariant} app`);
+    }
+  }, [copyVariant])
+
+  return (
+    <div>
+      <h1>{mainCopy}</h1>
+      <p>This is a page being A/B/n tested</p>
+    </div>
+  )
+}
+```
+
+When we reload our page, we get a title based on the flag key variant. 
+
+![Epic webpage](../images/tutorials/abn-testing/epic.png)
+
+### Checking the other variants
+
+We can check the other variants by going to the [feature flag page](https://app.posthog.com/feature_flags), searching for the key related to our experiment, and clicking on it to edit it. Scroll down to release conditions and set the **optional override** to any of the flag values. If you want the optional override to apply to only you, you can create another condition set. Finally, press save and when you go back to your app, you see the overridden flag value.
+
+![Overriding our variant](../images/tutorials/abn-testing/override.png)
+
+> **Note:** Make sure to remove the optional override before rolling out your A/B/n test to real users.
+
+### Rolling out the winning variant
+
+Once your experiment reaches significance, you can stop it and roll out the winning variant. 
```

Comment:
I wouldn't say significance, since this is a new concept that we havent introduced yet in this tuotiral.
```suggestion
Once your experiment has concluded, you can stop it and roll out the winning variant. 
```